doc/src/build/devnet.md

```diff
@@ -47,24 +47,35 @@ To connect the Sui client to Sui Devnet, run the following command:
 ```shell
 sui client
 ```
+
 The first time you start Sui client, it displays the following message:
 
-```shell
-Config file ["/Users/dir/.sui/sui_config/client.yaml"] doesn't exist, do you want to connect to a Sui RPC server [y/n]?
 ```
+Config file ["/Users/dir/.sui/sui_config/client.yaml"] doesn't exist, do you want to connect to a Sui full node server [y/n]?
+```
+
 Press **y** and then press **Enter**. It then asks for the RPC server URL: 
 
-```shell
-Sui RPC server Url (Default to Sui Devnet if not specified) :
 ```
-Press **Enter** to connect to Sui Devnet. To use a custom RPC server, enter the URL to the RPC endpoint to use.
+Sui full node server url (Default to Sui Devnet if not specified) :
+```
 
-```shell
-Select key scheme to generate keypair (0 for ed25519, 1 for secp256k1):
+Press **Enter** to connect to Sui Devnet. To use a custom RPC server, enter the URL to the custom RPC endpoint and then press **Enter**.
+
+```
+Select key scheme to generate keypair (0 for ed25519, 1 for secp256k1, 2: for secp256r1:
```

Comment:
```suggestion
Select key scheme to generate keypair (0 for ed25519, 1 for secp256k1, 2 for secp256r1:
```
This might be an issue with console output

Reply:
yep, missing the closing ) in console outpute

Reply:
https://github.com/MystenLabs/sui/blob/1baf0530d2e37916be85dd85e8db155a9559e9dd/crates/sui/src/sui_commands.rs#L490